Updated external references (parameters.yml and responses.yml) in prep for next sprint's work

digitaljoshua opened the pull request:
Added a few parameters, added a few response codes

GiselleLeeOSI commented on content/developer-guide/dev-guide-introduction.md:
```diff
@@ -3,4 +3,4 @@ uid: intro-dev-guide
 
-OSIsoft Cloud Services developer guide
\ No newline at end of file
+OSIsoft Cloud Services developer guide. This is what will appear for users when they click to the developer guide.
```
Hi Josh, this content is no longer valid. See if you can delete this file from your PR. If unsure, talk to me.

digitaljoshua replied:
i deleted this file from the PR (I think).

GiselleLeeOSI commented on content/external-references/parameters.yaml:
```diff
@@ -6,18 +6,46 @@ tenantId:
+  type: text/plain
+  value: |
+    Only return clients that have these tags
 query:
```
query is tricky. There are two kinds: the one that is not supported and the other that is. And the unsupported one is already documented later called query-community. So you shouldn't change the description of this one here.

digitaljoshua replied:
reverted back to original definitions; created new query-identity, count-identity, skip-identity

GiselleLeeOSI commented on content/external-references/parameters.yaml:
```diff
@@ -6,18 +6,46 @@ tenantId:
-    Query identifier
+    (Not supported) Query identifier
 count:
   type: text/plain
```
Same for count and  and skip. You should keep the original definitions (these are actively being used). Instead you should create count and skip as they apply to identity and label them count-identity(or count-client) and skip-identity(skip-client) respectively.

digitaljoshua replied:
reverted back to original definitions; created new query-identity, count-identity, skip-identity

GiselleLeeOSI reviewed (changes requested):
1. Please take a look my comments before making changes. 
2. Please find way to uninclude dev-guide-introduction.md from this pr.

digitaljoshua commented:
not sure if i made a mess of things. 

1. I removed the file content/developer-guide/dev-guide-introduction.md from this PR (hopefully I didn't remove it from the actual branch- I don't think I did but the messages for Git are a bit ambiguous)
2. Changed the modified definition of query back to its original verbiage. I'm sure there is a quick way of selecting that particular change in this PR and poof...cutting that out of the PR. But I just couldn't find it. So I copy/pasted the definition back to its original verbiage that way.
3. Ditto for definition of count
4. Ditto for definition of skip
5. Added 3 new parameters: query-identity, query-count, query-skip

GiselleLeeOSI commented on content/external-references/parameters.yaml:
```diff
@@ -30,6 +70,34 @@ identityProviderId:
+clientcredentialclientCount:
+  type: text/plain
+  value: |
+    Max number of roles to return
```
Between 82 and 96, don't you want to be consistent in your use of Max vs maximum?

digitaljoshua replied:
i fixed this
